### ISSUE TITLE
Makes all keyboard shortcuts application global

### ISSUE THREAD

**Issue by lfcnassif** [sepinf-inc/IPED#1268]
Most of them, if not all, are specific to some panels. #1266 used a global approach that could be used everywhere.

**Comment by tc-wleite** [comment]
I am including CTRL+B (show bookmark manager dialog) in the "global" shortcut handling function in the same PR of #1866.

I think that the other existing "local" shortcuts (pre-defined or assigned to bookmarks) should work only if a certain component is focused (like the main items table or the gallery), like it is today. So I am not sure if we should move them too.
Is there any other existing shortcut that could be "global"?  

About the "assigned to bookmarks" shortcuts, some users asked me to make them work in "related items" tables (e.g. parent, subitems). This would make a bit more sense after #1866. So I will open a new issue after finishing #1866, so we can discuss if it is worth changing the current behavior.

**Comment by lfcnassif** [comment]
Hi @tc-wleite! I agree bookmark related shortcuts should work just if an item is selected in the focused component (the viewer sometimes takes the focus unintentionally, that may be a problem...). But does that avoid centralizing them in a central manager? I think that may make shortcut management easier and could allow checking for shortcut conflicts, warning the user if needed. Is it possible to detect the current focused component in the application? Or to track the focus history (maybe manually) to workaround the viewer borrowing focus?

**Comment by lfcnassif** [comment]
PS: I know focus control is very tricky...

**Comment by tc-wleite** [comment]
I see. So it is a bit more complicated than I anticipated.
Anyway, for now I will keep the change I made to CTRL+B, which was straightforward and should be helpful in any scenario, and leave this issue open for now.
